### PR TITLE
Put the new endpoint configuration in the correct place

### PR DESCRIPTION
**Pull request by dkurepa**
https://github.com/dotnet/arcade-services/issues/2783
The original PR (https://github.com/dotnet/arcade-services/pull/2948) wasn't correct. I added the endpoint in the `Override` section, but there was nothing to override, which broke the deployment https://dev.azure.com/dnceng/internal/_build/results?buildId=2288375&view=logs&j=b277f1ae-44e7-5ce4-4818-00049b2c7654&t=300ea8c8-976b-58c1-6a89-17701eb43612&l=87

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

### Release Note Description
Not release note worthy

### CODE REVIEW

**Review by premun — COMMENTED** [review]
Does this turn on HTTP on this port for all envs?

File: src/Maestro/Maestro.Web/PackageRoot/ServiceManifest.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
            listen. Please note that if your service is partitioned, this port is shared with 
            replicas of different partitions that are placed in your code. -->
       <Endpoint Protocol="http" Name="ServiceEndpoint" Type="Input" Port="8080" />
-      <Endpoint Name="ServiceEndpointHttp" Port="8088" Protocol="http"/>
+      <Endpoint Name="ServiceEndpointHttp" Port="8088" Protocol="http" Type="Input" />
```
Comment (premun) [inline src/Maestro/Maestro.Web/PackageRoot/ServiceManifest.xml:38]:
```suggestion
      <Endpoint Name="ServiceEndpointHttp" Port="8088" Type="Input" Protocol="http" />
```

Comment (premun) [reply src/Maestro/Maestro.Web/PackageRoot/ServiceManifest.xml:38]: JUst so that it is the same as the line above

**Comment by dkurepa** [comment]
> Does this turn on HTTP on this port for all envs?

Yes, it does

**Comment by premun** [comment]
@dkurepa is this possible you F5 Maestro locally to see if this works?

The port should also be open on local right?

**Review by premun — APPROVED** [review]

**Review by premun — APPROVED** [review]

**Pull request merged**